website/content/ja/contribute/projects/translations/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 翻訳
+list_pages: false
+---
+
+## 導入
+
+CNCFの素晴らしさの一端は関わっている人々の多様性とコミュニティからサポートされているからです。このコミュニティをより包括的にする一例として、可能な箇所でコンテンツの翻訳を提供することです。翻訳には当然それを作成およびメンテナンスする母語話者を必要とし、このページはTAG App Deliveryのコンテンツでのやり方を示します。
```

Comment:
```suggestion
CNCFの素晴らしさの一端は関わっている人々の多様性とコミュニティからサポートされているからです。このコミュニティをより包括的にする一例として、可能な箇所でコンテンツの翻訳を提供することです。翻訳にはもちろん、ネイティブスピーカーが作成およびメンテナンスする必要があります。このページはTAG App Deliveryのコンテンツでのやり方を示します。
```

IMO: I think it would be better to split the sentence starting with "Translations of …" into two sentences.